kitsune/users/views.py

```diff
@@ -772,3 +776,41 @@ def validate_field(request):
                 }
 
     return data
+
+
+class FXAAuthenticateView(OIDCAuthenticationRequestView):
+
+    @staticmethod
+    def get_settings(attr, *args):
+        """Override settings for Firefox Accounts.
+
+        The default values for the OIDC lib are used for the SSO login in the admin
+        interface. For Firefox Accounts we need to pass different values, pointing to the
+        correct endpoints and RP specific attributes.
+        """
+
+        FXA_CONF = {
```

Comment:
I suggest you deduplicate this to a helper method since you are pretty much using the same settings.